docs/snippets/README.md

```diff
@@ -1,3 +1,5 @@
 # WooCommerce code snippets
 
 Various code snippets you can add to your site to enable various custom functionality.
+
+-   [Add a message above the login / register form](./before-login--register-form.md)
```

Comment:
I was thinking of adding sub headlines to the file, e.g. Frontend Code Snippets (see https://github.com/woocommerce/woodocs/issues/41). Given that it's only one code snippet now and that it's unknown how many code snippets will be added, I decided against it.

Reply:
Yup good call. We can add sub-headings as needed.